src/GraphQL.ApiTests/GraphQL.approved.txt

```diff
@@ -1918,8 +1920,7 @@ namespace GraphQL.Types
     }
     public interface IProvideResolvedType
     {
-        GraphQL.Types.IGraphType ResolvedType { get; }
-        System.Type Type { get; }
```

Comment:
Was not used. In addition, the name of the interface hints that this property shoul not be here.

Reply:
No breaking change aslo since there is no `IProvideResolvedType.Type` property in `master`.

Reply:
I would prefer to leave it here so the type, when available, can be examined easily.  But it will not _always_ be present and is not required at runtime, so it makes sense to remove it if you really think we should.

Reply:
I would like to remove it from here. This property is associated with another problem. It does not necessarily correspond to the value in `ResolvedType` (sometimes absolutely does not match as if the scalars are replaced).

Reply:
Ok